src/main/java/com/razorpay/ApiClient.java

```diff
@@ -31,27 +31,45 @@ class ApiClient {
 
   private final int STATUS_MULTIPLE_CHOICE = 300;
 
+  ApiClient() { }
```

Comment:
why is this added?

Reply:
I think its added for Oauth public APIs where we dont need to set any Auth.